### DISCUSSION TITLE
401 unauthorized- dapp down! AND cannot log in to cloud.walletconnect

### DISCUSSION THREAD

**Discussion by mintbet7** [WalletConnect/walletconnect-monorepo#3659] [Developer Support]
what the heck, is everything down? I don't know if this is some kind of issue with my account or what! my front-end cannot be used and we just launched our MVP!!

**Comment by mintbet7** [comment]
I should add 'https://cloud.walletconnect.com/sign-in' is not loading, I have no way to check my account. please email me asap, the associated email address is YUVA67V6@proton.me

**Reply by mintbet7** [reply]
this is what I see at cloud.walletconnect in the console.log: 

Failed to load resource: net::ERR_BLOCKED_BY_CLIENT
awyuaxlpezyhfkpznrox.supabase.co/rest/v1/rpc/is_current_user_admin:1     Failed to load resource: the server responded with a status of 401 ()
o1095249.ingest.sentry.io/api/6114425/envelope/?sentry_key=a9657c2f33bb46ff8df7af75f19e895f&sentry_version=7&sentry_client=sentry.javascript.nextjs%2F7.28.0:1     Failed to load resource: net::ERR_BLOCKED_BY_CLIENT
o1095249.ingest.sentry.io/api/6114425/envelope/?sentry_key=a9657c2f33bb46ff8df7af75f19e895f&sentry_version=7&sentry_client=sentry.javascript.nextjs%2F7.28.0:1     Failed to load resource: net::ERR_BLOCKED_BY_CLIENT
instrument.js:108 Object Objectcontext: "core/verify-api"[[Prototype]]: Object 'Verify iframe failed to load: https://verify.walletconnect.org'
(anonymous) @ instrument.js:108
instrument.js:108 Object Object 'verify iframe load timeout'
(anonymous) @ instrument.js:108
baseclient.js:669 [Violation] 'visibilitychange' handler took 878ms
client.js:122     POST https://o1095249.ingest.sentry.io/api/6114425/envelope/?sentry_key=a9657c2f33bb46ff8df7af75f19e895f&sentry_version=7&sentry_client=sentry.javascript.nextjs%2F7.28.0 net::ERR_BLOCKED_BY_CLIENT

**Reply by mintbet7** [reply]
please @glitch-txs

**Reply by mintbet7** [reply]
what the heck! what the heck! this is not okay!

**Reply by glitch-txs** [reply]
We'll take a look, thanks for reporting

**Reply by glitch-txs** [reply]
should be fixed by now!

**Reply by mintbet7** [reply]
what was the cause? Is this going to happen again?
